api/v1/swagger.yaml

```diff
@@ -2040,7 +2040,6 @@ paths:
           in: formData
           description: The unique readable name for the bot
           type: string
-          required: true
```

Comment:
careful! you don't want to change the bot stuff in this pr!

Reply:
Oops I haven't change anything. Maybe because I pulled and pushed. This is strange.